public/src/components/student/GroupInfo.tsx

```diff
@@ -1,22 +1,22 @@
 import * as React from "react";
 
-import { ICourse, ICourseGroup } from "../../models";
+import { ICourse } from "../../models";
 import { Group } from "../../../proto/ag_pb";
 
 interface IGroupPro {
-    group: ICourseGroup;
+    group: Group;
     course: ICourse;
 }
 class GroupInfo extends React.Component<IGroupPro, any> {
     public render() {
         const groupMembers: JSX.Element[] = [];
-        for (let i: number = 0; i < this.props.group.users.length; i++) {
-            groupMembers.push(<li key={i} className="list-group-item">{this.props.group.users[i].getName()}</li>);
+        for (let i: number = 0; i < this.props.group.getUsersList().length; i++) {
```

Comment:
Here I suggest to call `this.props.group.getUsersList()` before the for loop and assign it to a variable `users` and instead use `users.length` and `users[i].getName()` in the for loop.